### PR TITLE
feat: remove support for deploy certain services

### PR DESCRIPTION
**Pull request by jLopezbarb**
# Proposed changes

Fixes DEV-445

This PR removes the support for deploying several stack services from an `okteto deploy` command. 

## How to validate

1. Run `okteto deploy api` and check that an error is thrown
1. Using a compose file with dependant services
1. Run just one of the services that depends on another

Example: 
```yaml
version: '3'
services:
  a:
    image: nginx
    container_name: a

  b:
    image: nginx
    container_name: b
    depends_on:
      - a

  c:
    image: nginx
    container_name: c
```

``` yaml
deploy:
  compose:
    - file: docker-compose.yml
      services:
        - b
```
## CLI Quality Reminders 🔧

For both authors and reviewers:

- Scrutinize for potential regressions
- Ensure key automated tests are in place
- Build the CLI and test using the validation steps
- Assess Developer Experience impact (log messages, performances, etc)
- If too broad, consider breaking into smaller PRs
- Adhere to our [code style](https://github.com/okteto/okteto/blob/master/docs/code-style.md) and [code review](https://github.com/okteto/okteto/blob/master/docs/code-review.md) guidelines

### CODE REVIEW

File: cmd/deploy/deploy.go [hunk 1]
```diff
@@ -72,19 +72,19 @@ type Options struct {
...
-	RunWithoutBash   bool
-	RunInRemote      bool
-	Wait             bool
-	ShowCTA          bool
+	ManifestPath          string
+	Name                  string
+	Namespace             string
+	K8sContext            string
+	Variables             []string
+	StackServicesToDeploy []string
```
Comment (andreafalzetti) [inline cmd/deploy/deploy.go:80]: why did you rename this to `StackServicesToDeploy`?

**Review by andreafalzetti — APPROVED** [review]

File: cmd/deploy/deploy.go (hunk 1 above)
Comment (jLopezbarb) [reply cmd/deploy/deploy.go:80]: `ServicesToDeploy` was used through different parts of the command and was confusing (at least for me) while validating the user input so I decided to make it more clear. Let me know if we should rename back to its name
